<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QDialog" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Station</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Source</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="sourceCombo"/>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>55</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Country</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="countryCombo"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Region</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QComboBox" name="regionCombo"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>City</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="cityCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="canselButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>sourceCombo</tabstop>
  <tabstop>countryCombo</tabstop>
  <tabstop>regionCombo</tabstop>
  <tabstop>cityCombo</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>canselButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>canselButton</sender>
   <signal>clicked()</signal>
   <receiver>Setting</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>Setting</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sourceCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Setting</receiver>
   <slot>sourceChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>countryCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Setting</receiver>
   <slot>countryChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>regionCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Setting</receiver>
   <slot>regionChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>Setting</receiver>
   <slot>okClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sourceChanged(int)</slot>
  <slot>countryChanged(int)</slot>
  <slot>regionChanged(int)</slot>
  <slot>okClicked()</slot>
 </slots>
</ui>
